src/test/java/org/broadinstitute/consent/http/service/dao/DatasetServiceDAOTest.java

```diff
@@ -725,6 +725,30 @@ void testPatchDataset() throws Exception {
     assertEquals(3, patched.getProperties().size());
   }
 
+  @Test
+  void testPatchDatasetWithNullName() throws Exception {
+    Dataset dataset = createDataset();
+    User user = userDAO.findUserById(dataset.getCreateUserId());
+    DatasetPatch patch = new DatasetPatch(null, List.of());
+    serviceDAO.patchDataset(dataset.getDatasetId(), user, patch);
+    Dataset patched = datasetDAO.findDatasetById(dataset.getDatasetId());
+
+    // Validate that the name is NOT updated to a null value
+    assertEquals(dataset.getName(), patched.getDatasetName());
+  }
+
+  @Test
+  void testPatchDatasetWithEmptyName() throws Exception {
+    Dataset dataset = createDataset();
+    User user = userDAO.findUserById(dataset.getCreateUserId());
+    DatasetPatch patch = new DatasetPatch("", List.of());
```

Comment:
Maybe it should contain whitespace to test `isBlank()`? Otherwise it's only testing the `isEmpty()` case.
```suggestion
    DatasetPatch patch = new DatasetPatch("    ", List.of());
```